ms_game_chat/backend/requirements.txt

```diff
@@ -3,5 +3,4 @@ psycopg[binary]
 channels~=4.0
 daphne~=4.0
 channels_redis~=4.1
-ft_jwt
-
+ft_jwt==0.1.0
```

Comment:
@mariekart42 without version number we did not get the newest version, had to hardcode now a specific release